### ISSUE TITLE
Trying to make hb_pb custom key as the CPM

### ISSUE THREAD

**Issue by MDG2600** [prebid/line-item-manager#46]
* line-item-manager version: 0.2.1
* Python version: 3.9
* Operating System: macOS

### Description
I'm doing a hybrid of single order and send all bids, to cut down on the amount of key-values in my GPT and keep reporting at the advertiser/order/line item level. 
My line item targeting for example in GAM:
placement=("Placement name") AND custom=("hb_bidder=ix AND hb_pb=0.01")

### What I Did
Here is the error I receive.
ERROR:line_item_manager.cli:Google Ads Error, [CustomTargetingError.KEY_NAME_INVALID_CHARS @ name; trigger:'<"{{ cpm }}">']

Basically I want to see if i can use the cpm of the line as a key-value.

[config.txt](https://github.com/prebid/line-item-manager/files/6085944/config.txt)

```
Paste the command(s) you ran and the output.  And if relevant, attach your YAML config file.
If there was a crash, please include the traceback here.
```

**Comment by MDG2600** [comment]
I can make it work the way I need once the custom key value change in PR #45 goes out.

**Comment by timbolimboslice** [comment]
@MDG260 Were you able to solve having the bidder name and price bucket as two separate key value pairs? I'm trying to do the same but keep running into the same error as you above.

Config:
```
bidder_key_map:
  medianet:
    hb_pb: "<{{ cpm }}>"
    hb_bidder: <medianet>
```

Current Results
<img width="826" alt="Screen Shot 2021-11-22 at 11 00 48 AM" src="https://user-images.githubusercontent.com/11657201/142903506-35ea4779-2afc-4bfe-9ba6-212c4141157d.png">
0
